src/numbers.js

```diff
@@ -1,45 +1,45 @@
 function add (a, b) {
-  // your code here
+  return a + b;
 }
 
 function subtract (a, b) {
-  // your code here
+  return a - b;
 }
 
 function multiply (a, b) {
-  // your code here
+  return a * b;
 }
 
 function divide (a, b) {
-  // your code here
+  return a / b;
 }
 
 function power (a, b) {
-  // your code here
+  return a ** b;
 }
 
 function round (a) {
-  // your code here
+  return Math.round(a);
 }
 
 function roundUp (a) {
-  // your code here
+  return Math.ceil(a);
 }
 
 function roundDown (a) {
-  // your code here
+  return Math.floor(a);
 }
 
 function absolute (a) {
-  // your code here
+  return Math.abs(a);
 }
 
 function quotient (a, b) {
-  // your code here
+	return Math.trunc(a / b);
 }
 
 function remainder (a, b) {
-  // your code here
+ return (a % b);
 }
```

Comment:
there shouldn't be a space between the function's name and the parenthesis, i.e. `function add(a, b) { ...` instead of `function add (a, b) { ...`